akka-projection-core/src/main/scala/akka/projection/TelemetrySpi.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
+
+import scala.collection.immutable
+
+import akka.actor.typed.ActorSystem
+import akka.annotation.ApiMayChange
+import akka.annotation.InternalApi
+import akka.annotation.InternalStableApi
+
+/**
+ * Service Provider Interface (SPI) for collecting metrics from projections.
+ *
+ * Implementations must include a single constructor with two arguments: [[ProjectionId]]
+ * and [[ActorSystem[_]].  To setup your implementation, add a setting on your `application.conf`:
+ *
+ * {{{
+ * akka.projection.telemetry.implementation-class = "com.example.MyMetrics"
+ * }}}
+ */
+@ApiMayChange trait Telemetry {
```

Comment:
Given that we have decided to keep Telemetry and StatusObserver separate I think we should take full advantage of that and move this back to `internal`. Can still be public class but in `internal` package and with the `@InternalStableApi` marker. That will make it easier for us to change the signatures in Telemetry in the future as needed.

Therefore I also revert my previous opinion about defining the implementation-class in reference.conf. Since it's no longer intended for users we should not show it there. Back to the `hasPath` as you had originally.